neural_search/util.py

```diff
@@ -198,3 +220,16 @@ def detect_outliers(
     outlier_scores = outlier_scores[sorted_index[:max_records]]
 
     return status.HTTP_200_OK, [outlier_ids.tolist(), outlier_scores.tolist()]
+
+
+def update_embedding_payload(project_id: str,embedding_id:str):
```

Comment:
again maybe wrong formatting  here?

Reply:
yes, but removed code, resolving the conversation